www/pages/guides/netlify-deploy.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Deploy on Netlify'
+menu: side
+linkheadings: 3
+index: 1
```

Comment:
AWS guide already has an index of 1

Reply:
@aholtzman 
This still seems to apply I think, so this and AWS guide need to figure out a new ordering

Reply:
Yeah - the AWS order was set before Netlify doc was created, but the Netlify should be higher based on importance to users.